sites/funcorporate/src/backend/funcorporate/settings.py

```diff
@@ -437,6 +437,7 @@ class Base(StyleguideMixin, DRFMixin, RichieCoursesConfigurationMixin, Configura
     RICHIE_ES_INDICES_PREFIX = values.Value(
         default="richie", environ_name="RICHIE_ES_INDICES_PREFIX", environ_prefix=None
     )
+    RICHIE_ES_STATE_WEIGHTS = values.ListValue(None)
 
```

Comment:
are you missing teaser placeholder changes?

Reply:
Whoops!